src/zenml/stack_stores/base_stack_store.py

```diff
@@ -157,6 +161,22 @@ def register_stack_component(
                 and name already exists.
         """
 
+    @abstractmethod
+    def update_stack_component(
+        self,
+        current_component: StackComponentWrapper,
```

Comment:
Since this is all unimplemented, Maybe consider leaving it out? But if it does stay in, I think the function signature should not take 2 StackComponentWrappers, but rather a type and name of the component to update, and a wrapper to update it with (same signature as the create stack component)